tests/Oryx.Integration.Tests/Php/PhpWordPressTest.cs

```diff
@@ -30,9 +30,7 @@ public PhpWordPressTest(ITestOutputHelper output, TestTempDirTestFixture fixture
         public async Task PipelineTestInvocationsPhp80Async()
         {
             string phpVersion80 = "8.0";
-            await Task.WhenAll(
```

Comment:
Wordpress 5.1 and php 8 do not seem compatible https://make.wordpress.org/core/handbook/references/php-compatibility-and-wordpress-versions/